app/routes.py

```diff
@@ -1,2 +1,93 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.planet import Planet
+from app import db
 
+planet_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+# POST /planets
+
+
+@planet_bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    try:
+        new_planet = Planet.from_dict(request_body)
+        db.session.add(new_planet)
+        db.session.commit()
+
+        message = jsonify(f"Planet {new_planet.name} successfully created")
+        return make_response(message, 201)
+
+    except KeyError as e:
+        message = jsonify(f"missing required value: {e}")
+        abort(make_response(message, 400))
```

Comment:
Nice job using try/except to add in error handling for a request that relies on (possibly incorrect) client data.